conf/messages

```diff
@@ -84,11 +84,28 @@ payment.noPayment = No payments due right now
 payment.nextPayment = Next payment due
 payment.nextPayments = Payments due
 payment.check = Check what you owe
+payment.afterSubmitted = After you have submitted a return, it can take 24 hours for what you owe to show here.
```

Comment:
Why have new English messages been added? Can all of the changes to this file be reverted please.